lib/easy_meli/errors.rb

```diff
@@ -73,4 +73,23 @@ def local_message
       'Unknown Error'
     end
   end
+
+  class ErrorParser
+    ERROR_LIST = {
+      'Error validating grant' => EasyMeli::InvalidGrantError,
+      'The User ID must match the consultant\'s' => EasyMeli::ForbiddenError,
+      'invalid_token' => EasyMeli::InvalidTokenError,
+      'Malformed access_token' => EasyMeli::MalformedTokenError,
+      'too_many_requests' => EasyMeli::TooManyRequestsError,
+      'unknown_error' => EasyMeli::UnknownError
+    }
+
+  def self.error_class(body)
```

Comment:
Bad indents but I would also move this into its own file. I think this file should be only for errors.